test/test_helper.rb

```diff
@@ -31,6 +32,10 @@ def region
     "region #{ENV['region']}" if ENV['region']
   end
 
+  def endpoint
+    "endpoint #{ENV['endpoint']}" if ENV['endpoint']
```

Comment:
nit:
Can we create stub AWS environment within CO environment with this method?

Reply:
> Can we create stub AWS within CI environment with this method?

Yes. With this patch applied, you can run the test suite against the mock
AWS server (provided you've set up something like below).

```bash
# Running a moto server as a background process
$ moto_server logs &

# Execute the unit test
$ export region=us-east-1
$ export endpoint="http://localhost:5000"
$ bundle exec rake test
```